### DISCUSSION TITLE
Detecting garage door open/closed status from camera image

### DISCUSSION THREAD

**Discussion by backslashV** [blakeblackshear/frigate#1666] [Ask A Question]
I was wondering if it's possible to detect the door's open/closed status with frigate. I have a camera pointed at the door and want to see if there's a way to analyze part of the picture to see if the door is open or closed.

**Comment by ElMoribond** [comment]
Unless using a specifically trained model, Frigate will never be able to meet this kind of need.
If you know a tensorflow model for this, indicate it.

Otherwise you can take a look [here](https://github.com/twstokes/door_detect).
